skbio/stats/distance/tests/test_base.py

```diff
@@ -979,6 +1010,55 @@ def test_to_series_default_ids(self):
         exp = pd.Series([0.123], index=pd.Index([('0', '1')]))
         assert_series_almost_equal(series, exp)
 
+    def test_validate_asym_shape(self):
+        # first check it actually likes good matrices
+        data_good = np.array([[0., 42.], [42., 0.]])
+        data_sym, data_hollow = is_symmetric_and_hollow(data_good)
+        self.assertEqual(data_sym, True)
+        del data_sym
```

Comment:
why all the `del`s?

Reply:
Mostly logical consistency...
Making sure I do not use an old value sometime later in the code.

Reply:
Ah, thanks :)